src/getCoinCombination.test.js

```diff
@@ -1,10 +1,49 @@
-'use strict';
+"use strict";
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+describe(`getCoinCombination`, () => {
+  const { getCoinCombination } = require("./getCoinCombination");
 
   it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it(`should be positive number`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+    expect(getCoinCombination(17)).toEqual([2, 1, 1, 0]);
+  });
```

Comment:
The description 'should be positive number' is misleading because the test is checking the function's output for various valid inputs, not just positive numbers. Consider renaming it to something like 'should return correct coin combination for valid inputs'.